<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Data_center</class>
 <widget class="QWidget" name="Data_center">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>698</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,9">
   <item>
    <widget class="QWidget" name="panel" native="true">
     <property name="styleSheet">
      <string notr="true">#panel {
background-color: rgb(238, 238, 236);
}

</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QCheckBox" name="checkBox_2">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>技术管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_2" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_2{
background-color: rgb(238, 238, 236);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <widget class="QPushButton" name="pro_plan">
           <property name="styleSheet">
            <string notr="true">/*background-color: #455364;
  border-bottom: 2px solid #455364;*/</string>
           </property>
           <property name="text">
            <string>技术方案</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="Innovation">
           <property name="text">
            <string>优化创新</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox_3">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>安全管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget_3" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget_3{
background-color: rgb(238, 238, 236);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_9">
         <item>
          <widget class="QPushButton" name="safe_rules">
           <property name="text">
            <string>安全细则</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="safe_logs">
           <property name="text">
            <string>安全检查记录</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="checkBox">
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>设备管理</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">#widget{
background-color: rgb(238, 238, 236);
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QPushButton" name="signup">
           <property name="text">
            <string>设备登记</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="logs">
           <property name="text">
            <string>设备使用情况</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_4">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page_plan"/>
     <widget class="QWidget" name="page_update"/>
     <widget class="QWidget" name="page_real"/>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
